test/jdk/javax/naming/spi/providers/InitialContextTest.java

```diff
@@ -243,9 +261,15 @@ private static boolean verifyOutput(String output, String fqn) {
 
     private static void jar(Path jarName, Path jarRoot) {
         String jar = getJDKTool("jar");
-        ProcessBuilder p = new ProcessBuilder(jar, "cf", jarName.toString(),
-                "-C", jarRoot.toString(), ".");
-        quickFail(run(p));
+        String [] commands = {jar, "cf", jarName.toString(),"-C", jarRoot.toString(), "."};
+        try {
+            OutputAnalyzer outputAnalyzer = ProcessTools.executeCommand(commands);
+            if(outputAnalyzer.getExitValue() != 0) {
+                throw new RuntimeException(outputAnalyzer.getOutput());
```

Comment:
I think it would be better to just call:
```java
outputAnalyzer.shouldHaveExitValue(0);
```
that way it even prints the stdout/stderr logs and throws the exception.